course/models.py

```diff
@@ -450,6 +450,11 @@ def __str__(self):
     def clean(self):
         super().clean()
         errors = {}
+        RESERVED = ("instances")
+        if self.instance_name in RESERVED:
+            errors['instance_name'] = _("You cannot use word '{}' as an instance name.").format(RESERVED)
```

Comment:
`RESERVED` could a tuple of many keywords. The `format` argument should be `self.instance_name` so that you include the user input in the error message.

Reply:
Good point!